topological_utils/scripts/LoadPointSet.py

```diff
@@ -7,57 +7,96 @@
 
 from topological_navigation.topological_node import *
 from strands_navigation_msgs.msg import TopologicalNode
+from strands_navigation_msgs.msg import TopologicalMap
+from strands_navigation_msgs.msg import TopMap
+from strands_navigation_msgs.msg import TopNode
+from strands_navigation_msgs.msg import TopEdge
```

Comment:
So, the format of the topo map on mongodb has changed, right? Is there a conversion script for others to use?